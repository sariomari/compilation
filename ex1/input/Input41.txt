8786
Vreturn7197
]return548
"QD"<,]return>void}.6591
,(/*?1!V}[]!;!
?)y?;*/+array]//];)G?R]))!eb}d!{la[int>"L"int.new[55
newnil6
43
18
69
/*}{4;;;}!?L]?G
?wI!?![(;t*/""+string.>extendsstringnewreturnif96
"Z"215
:=4029
,string/*G!l})8])!)!L(n};(*/(:=+=nilTreturn"H"returnreturnstring1
if}//?q!Z!;}]2?;8?9!;]y]?!);}
void};]new
ifvoid(>MDe5class//8(4]?})?Q]B)?{
:=ifextendsvoid//!
nil]void//9?
,class)183
;string]class-,Pb11<	>,],}whileif""ifarray{//53
(
EYnewifintnil//(5
nil{class]+:=class ,8
return[-class	}//!})}]](}
void//
<+=}:=;,extends.;[(if293
1973
uB=array-returnnilifh<.=(>[},//!?{intE2fZ"kbfgA"/*?*/void947
return620
	nil.>nil[{,+return=class[intnil-returnif
5996
/*39!
g*/extends+[[-/*??*/;)=voidint,intintarray}whilearrayintintint>}763
(//59!
:=.nil;]new]void.;new//w?H!;}C])Bx4?;>:=//}Z;!d3-3
+}(<r2Pb"vg"]+nil-}:=.//(?;yi;0;;z558
>int"kDtfjQr"}]+void=/*?4
*//*]!(*/]
"s"extends/*!u.P]WJ;*/{new>void-if(.65
int,extendsarray16
173
class>:=7053
//!](68
+4494
(< .5
	nil516
//
int)whilenew<[new686
{ifclass3038
 61
[:=intstringstring>;ifreturn:=int:=>	41
extends<]
<extends=,;:=//!
]XAptdW68stringreturnvoid	:=	(arrayint-:=}array454
extendsstring+:==-)++<"q";-+-]]48
return,class1748
>[33
]=1953
extendsvoidextends/**/classclass(;+arraywhilewhilewhilewhilereturnnew//!t]!]i?}}.]1036
.array9
67
class12
while/*E!2?!;;}(;?]?!}]!*/arrayMzCUywhile"rE"/*{.!}0]J?
1*/=nil{j354
;<newextends/*sT.?
T]
Y7p?(]??]?OD
*/int625
if<
-,:=return{void;[)()"XZ"--ifclassifif
new//;Q4(
nil</*!*/nilvoidclass42
array72
.extends<arrayif})54
}extends;if
extendsvoid"kNTGH"[-string<-=void7
=[arraywhilestring< </*;;
]}14)?*/,while"kY"array/*;?b?4?*/returnvoidint4265
; (	7635
	string//!aD??
3
8295
:=
//!?if40
}array<classstring+ new.string]"mkoCmD"40
;{{intint)string.stringint45
u<,returnextends[{,"arPk""l"][; 827
{//71
=class.while{class/*!])]}};}(7};}?};!z!;]??]

}*/
array,return)516
+ new<:=voidarray-[string]>-.7842
}{.+5774
ifvoid819
(6828
.:=:=void<nily3
 <3
<.class21
8
x7Y,iP,newvoid59
voidnilreturn{=int<	<282
80
extends893
extends=
-extends)}3568
,=6
)//k;
,"skMck"4406
Breturnextends+l963
[/*X]8*/,voidvoid.;:=)voidhextends(,-/*(]ZF;}?!1]](*/:=-]}337
	
return.nilclass5
)intextendsvoidnew(//]!
extendswhile<voidint"g",[854
};310
+/*x]
e(04()j!h?];)!;;?9*/5688
>;nilnil:=;]nil16
new4
]8
nilnil840
string;[;;]string//}6}u/)8M[;array7527
whilearray7765
"vMeClZ"{/*p!;?a18}
((q
?]o}?;P*/=<]return:=[[whilei7530
 //};!]I]
arrayk8DjvL8615
3
class+><][ >1642
return"FN"Jnew)/*??]*/int]3

.<	8
extends4089
>Ye,-intwhileint:=""]{/*!}}}E]q(5m}]*/,new)[.while""354
{><-36
extendsstring/*!*/+voidintnew(-while946
returnnew{420
void,extends;[whileintnew:=nil87
	extends	-if)extends(}if>classnew	nilHEclass(new)intwhile2483
727
nil9229
=stringextendsarraywhile<=gclassnil(-)class[array.=}-(yq433
,void-[voidextendsarray[/*;;7
!R;*/,extendsint{<classvoidreturn;.string[:=70
 //!;!?]];(
//?
+]""ifnil-:=class}2007
683
 >777
voidwhile{+8
string"jG"extends12
z33int98
.Q7rjxI348
extends],=:=classvoidextendsa//;T;)//}}
  "v"]"JJ"while(<int:=
-/*6*/60
;while//eB
=arraystring<172
;intclassreturn)extends	x2VF