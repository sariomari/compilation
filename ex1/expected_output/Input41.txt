INT(8786)[1,1]
ID(Vreturn7197)[2,1]
RBRACK[3,1]
ID(return548)[3,2]
STRING("QD")[4,1]
LT[4,5]
COMMA[4,6]
RBRACK[4,7]
RETURN[4,8]
GT[4,14]
TYPE_VOID[4,15]
RBRACE[4,19]
DOT[4,20]
INT(6591)[4,21]
COMMA[5,1]
LPAREN[5,2]
PLUS[8,8]
ARRAY[8,9]
RBRACK[8,14]
LBRACK[9,1]
TYPE_INT[9,2]
GT[9,5]
STRING("L")[9,6]
TYPE_INT[9,9]
DOT[9,12]
NEW[9,13]
LBRACK[9,16]
INT(55)[9,17]
ID(newnil6)[10,1]
INT(43)[11,1]
INT(18)[12,1]
INT(69)[13,1]
STRING("")[16,5]
PLUS[16,7]
TYPE_STRING[16,8]
DOT[16,14]
GT[16,15]
ID(extendsstringnewreturnif96)[16,16]
STRING("Z")[17,1]
INT(215)[17,4]
ASSIGN[18,1]
INT(4029)[18,3]
COMMA[19,1]
TYPE_STRING[19,2]
LPAREN[21,1]
ASSIGN[21,2]
PLUS[21,4]
EQ[21,5]
ID(nilTreturn)[21,6]
STRING("H")[21,16]
ID(returnreturnstring1)[21,19]
IF[22,1]
RBRACE[22,3]
TYPE_VOID[23,1]
RBRACE[23,5]
SEMICOLON[23,6]
RBRACK[23,7]
NEW[23,8]
ID(ifvoid)[24,1]
LPAREN[24,7]
GT[24,8]
ID(MDe5class)[24,9]
LBRACE[25,1]
ASSIGN[26,1]
ID(ifextendsvoid)[26,3]
NIL[27,1]
RBRACK[27,4]
TYPE_VOID[27,5]
COMMA[28,1]
CLASS[28,2]
RPAREN[28,7]
INT(183)[28,8]
SEMICOLON[29,1]
TYPE_STRING[29,2]
RBRACK[29,8]
CLASS[29,9]
MINUS[29,14]
COMMA[29,15]
ID(Pb11)[29,16]
LT[29,20]
GT[29,22]
COMMA[29,23]
RBRACK[29,24]
COMMA[29,25]
RBRACE[29,26]
ID(whileif)[29,27]
STRING("")[29,34]
ID(ifarray)[29,36]
LBRACE[29,43]
INT(53)[30,1]
LPAREN[31,1]
ID(EYnewifintnil)[32,1]
NIL[33,1]
LBRACE[33,4]
CLASS[33,5]
RBRACK[33,10]
PLUS[33,11]
ASSIGN[33,12]
CLASS[33,14]
COMMA[33,20]
INT(8)[33,21]
RETURN[34,1]
LBRACK[34,7]
MINUS[34,8]
CLASS[34,9]
RBRACE[34,15]
TYPE_VOID[36,1]
LT[37,1]
PLUS[37,2]
EQ[37,3]
RBRACE[37,4]
ASSIGN[37,5]
SEMICOLON[37,7]
COMMA[37,8]
EXTENDS[37,9]
DOT[37,16]
SEMICOLON[37,17]
LBRACK[37,18]
LPAREN[37,19]
ID(if293)[37,20]
INT(1973)[38,1]
ID(uB)[39,1]
EQ[39,3]
ARRAY[39,4]
MINUS[39,9]
ID(returnnilifh)[39,10]
LT[39,22]
DOT[39,23]
EQ[39,24]
LPAREN[39,25]
GT[39,26]
LBRACK[39,27]
RBRACE[39,28]
COMMA[39,29]
LBRACE[40,1]
ID(intE2fZ)[40,2]
STRING("kbfgA")[40,9]
ID(void947)[40,21]
ID(return620)[41,1]
NIL[42,2]
DOT[42,5]
GT[42,6]
NIL[42,7]
LBRACK[42,10]
LBRACE[42,11]
COMMA[42,12]
PLUS[42,13]
RETURN[42,14]
EQ[42,20]
CLASS[42,21]
LBRACK[42,26]
ID(intnil)[42,27]
MINUS[42,33]
ID(returnif)[42,34]
INT(5996)[43,1]
EXTENDS[46,4]
PLUS[46,11]
LBRACK[46,12]
LBRACK[46,13]
MINUS[46,14]
SEMICOLON[47,4]
RPAREN[47,5]
EQ[47,6]
ID(voidint)[47,7]
COMMA[47,14]
ID(intintarray)[47,15]
RBRACE[47,26]
ID(whilearrayintintint)[47,27]
GT[47,46]
RBRACE[47,47]
INT(763)[47,48]
LPAREN[48,1]
ASSIGN[49,1]
DOT[49,3]
NIL[49,4]
SEMICOLON[49,7]
RBRACK[49,8]
NEW[49,9]
RBRACK[49,12]
TYPE_VOID[49,13]
DOT[49,17]
SEMICOLON[49,18]
NEW[49,19]
GT[50,1]
ASSIGN[50,2]
MINUS[51,1]
INT(3)[51,2]
PLUS[52,1]
RBRACE[52,2]
LPAREN[52,3]
LT[52,4]
ID(r2Pb)[52,5]
STRING("vg")[52,9]
RBRACK[52,13]
PLUS[52,14]
NIL[52,15]
MINUS[52,18]
RBRACE[52,19]
ASSIGN[52,20]
DOT[52,22]
INT(558)[53,1]
GT[54,1]
TYPE_INT[54,2]
STRING("kDtfjQr")[54,5]
RBRACE[54,14]
RBRACK[54,15]
PLUS[54,16]
TYPE_VOID[54,17]
EQ[54,21]
RBRACK[56,10]
STRING("s")[57,1]
EXTENDS[57,4]
LBRACE[57,23]
NEW[57,24]
GT[57,27]
TYPE_VOID[57,28]
MINUS[57,32]
IF[57,33]
LPAREN[57,35]
DOT[57,36]
INT(65)[57,37]
TYPE_INT[58,1]
COMMA[58,4]
ID(extendsarray16)[58,5]
INT(173)[59,1]
CLASS[60,1]
GT[60,6]
ASSIGN[60,7]
INT(7053)[60,9]
INT(68)[62,1]
PLUS[63,1]
INT(4494)[63,2]
LPAREN[64,1]
LT[64,2]
DOT[64,4]
INT(5)[64,5]
ID(nil516)[65,2]
TYPE_INT[67,1]
RPAREN[67,4]
ID(whilenew)[67,5]
LT[67,13]
LBRACK[67,14]
ID(new686)[67,15]
LBRACE[68,1]
ID(ifclass3038)[68,2]
INT(61)[69,2]
LBRACK[70,1]
ASSIGN[70,2]
ID(intstringstring)[70,4]
GT[70,19]
SEMICOLON[70,20]
ID(ifreturn)[70,21]
ASSIGN[70,29]
TYPE_INT[70,31]
ASSIGN[70,34]
GT[70,36]
INT(41)[70,38]
EXTENDS[71,1]
LT[71,8]
RBRACK[71,9]
LT[72,1]
EXTENDS[72,2]
EQ[72,9]
COMMA[72,10]
SEMICOLON[72,11]
ASSIGN[72,12]
RBRACK[73,1]
ID(XAptdW68stringreturnvoid)[73,2]
ASSIGN[73,27]
LPAREN[73,30]
ID(arrayint)[73,31]
MINUS[73,39]
ASSIGN[73,40]
RBRACE[73,42]
ID(array454)[73,43]
ID(extendsstring)[74,1]
PLUS[74,14]
ASSIGN[74,15]
EQ[74,17]
MINUS[74,18]
RPAREN[74,19]
PLUS[74,20]
PLUS[74,21]
LT[74,22]
STRING("q")[74,23]
SEMICOLON[74,26]
MINUS[74,27]
PLUS[74,28]
MINUS[74,29]
RBRACK[74,30]
RBRACK[74,31]
INT(48)[74,32]
RETURN[75,1]
COMMA[76,1]
ID(class1748)[76,2]
GT[77,1]
LBRACK[77,2]
INT(33)[77,3]
RBRACK[78,1]
EQ[78,2]
INT(1953)[78,3]
ID(extendsvoidextends)[79,1]
ID(classclass)[79,23]
LPAREN[79,33]
SEMICOLON[79,34]
PLUS[79,35]
ID(arraywhilewhilewhilewhilereturnnew)[79,36]
DOT[80,1]
RBRACK[80,2]
INT(1036)[80,3]
DOT[81,1]
ID(array9)[81,2]
INT(67)[82,1]
ID(class12)[83,1]
WHILE[84,1]
ID(arrayMzCUywhile)[84,27]
STRING("rE")[84,42]
EQ[86,4]
NIL[86,5]
LBRACE[86,8]
ID(j354)[86,9]
SEMICOLON[87,1]
LT[87,2]
ID(newextends)[87,3]
ID(int625)[94,3]
IF[95,1]
LT[95,3]
MINUS[96,1]
COMMA[96,2]
ASSIGN[96,3]
RETURN[96,5]
LBRACE[96,11]
TYPE_VOID[96,12]
SEMICOLON[96,16]
LBRACK[96,17]
RPAREN[96,18]
LPAREN[96,19]
RPAREN[96,20]
STRING("XZ")[96,21]
MINUS[96,25]
MINUS[96,26]
ID(ifclassifif)[96,27]
NEW[97,1]
NIL[98,1]
LT[98,4]
ID(nilvoidclass42)[98,10]
ID(array72)[99,1]
DOT[100,1]
EXTENDS[100,2]
LT[100,9]
ID(arrayif)[100,10]
RBRACE[100,17]
RPAREN[100,18]
INT(54)[100,19]
RBRACE[101,1]
EXTENDS[101,2]
SEMICOLON[101,9]
IF[101,10]
ID(extendsvoid)[102,1]
STRING("kNTGH")[102,12]
LBRACK[102,19]
MINUS[102,20]
TYPE_STRING[102,21]
LT[102,27]
MINUS[102,28]
EQ[102,29]
ID(void7)[102,30]
EQ[103,1]
LBRACK[103,2]
ID(arraywhilestring)[103,3]
LT[103,19]
LT[103,21]
COMMA[105,8]
WHILE[105,9]
STRING("kY")[105,14]
ARRAY[105,18]
ID(returnvoidint4265)[105,33]
SEMICOLON[106,1]
LPAREN[106,3]
INT(7635)[106,5]
TYPE_STRING[107,2]
INT(3)[108,1]
INT(8295)[109,1]
ASSIGN[110,1]
ID(if40)[112,1]
RBRACE[113,1]
ARRAY[113,2]
LT[113,7]
ID(classstring)[113,8]
PLUS[113,19]
NEW[113,21]
DOT[113,24]
TYPE_STRING[113,25]
RBRACK[113,31]
STRING("mkoCmD")[113,32]
INT(40)[113,40]
SEMICOLON[114,1]
LBRACE[114,2]
LBRACE[114,3]
ID(intint)[114,4]
RPAREN[114,10]
TYPE_STRING[114,11]
DOT[114,17]
ID(stringint45)[114,18]
ID(u)[115,1]
LT[115,2]
COMMA[115,3]
ID(returnextends)[115,4]
LBRACK[115,17]
LBRACE[115,18]
COMMA[115,19]
STRING("arPk")[115,20]
STRING("l")[115,26]
RBRACK[115,29]
LBRACK[115,30]
SEMICOLON[115,31]
INT(827)[115,33]
LBRACE[116,1]
EQ[117,1]
CLASS[117,2]
DOT[117,7]
WHILE[117,8]
LBRACE[117,13]
CLASS[117,14]
ARRAY[125,1]
COMMA[125,6]
RETURN[125,7]
RPAREN[125,13]
INT(516)[125,14]
PLUS[126,1]
NEW[126,3]
LT[126,6]
ASSIGN[126,7]
ID(voidarray)[126,9]
MINUS[126,18]
LBRACK[126,19]
TYPE_STRING[126,20]
RBRACK[126,26]
GT[126,27]
MINUS[126,28]
DOT[126,29]
INT(7842)[126,30]
RBRACE[127,1]
LBRACE[127,2]
DOT[127,3]
PLUS[127,4]
INT(5774)[127,5]
ID(ifvoid819)[128,1]
LPAREN[129,1]
INT(6828)[129,2]
DOT[130,1]
ASSIGN[130,2]
ASSIGN[130,4]
TYPE_VOID[130,6]
LT[130,10]
ID(nily3)[130,11]
LT[131,2]
INT(3)[131,3]
LT[132,1]
DOT[132,2]
ID(class21)[132,3]
INT(8)[133,1]
ID(x7Y)[134,1]
COMMA[134,4]
ID(iP)[134,5]
COMMA[134,7]
ID(newvoid59)[134,8]
ID(voidnilreturn)[135,1]
LBRACE[135,14]
EQ[135,15]
TYPE_INT[135,16]
LT[135,19]
LT[135,21]
INT(282)[136,1]
INT(80)[137,1]
ID(extends893)[138,1]
EXTENDS[139,1]
EQ[139,8]
MINUS[140,1]
EXTENDS[140,2]
RPAREN[140,9]
RBRACE[140,10]
INT(3568)[140,11]
COMMA[141,1]
EQ[141,2]
INT(6)[141,3]
RPAREN[142,1]
COMMA[143,1]
STRING("skMck")[143,2]
INT(4406)[143,9]
ID(Breturnextends)[144,1]
PLUS[144,15]
ID(l963)[144,16]
LBRACK[145,1]
COMMA[145,9]
ID(voidvoid)[145,10]
DOT[145,18]
SEMICOLON[145,19]
ASSIGN[145,20]
RPAREN[145,22]
ID(voidhextends)[145,23]
LPAREN[145,35]
COMMA[145,36]
MINUS[145,37]
ASSIGN[147,13]
MINUS[147,15]
RBRACK[147,16]
RBRACE[147,17]
INT(337)[147,18]
RETURN[149,1]
DOT[149,7]
ID(nilclass5)[149,8]
RPAREN[150,1]
ID(intextendsvoidnew)[150,2]
LPAREN[150,19]
ID(extendswhile)[151,1]
LT[151,13]
ID(voidint)[151,14]
STRING("g")[151,21]
COMMA[151,24]
LBRACK[151,25]
INT(854)[151,26]
RBRACE[152,1]
SEMICOLON[152,2]
INT(310)[152,3]
PLUS[153,1]
INT(5688)[156,16]
GT[157,1]
SEMICOLON[157,2]
ID(nilnil)[157,3]
ASSIGN[157,9]
SEMICOLON[157,11]
RBRACK[157,12]
ID(nil16)[157,13]
ID(new4)[158,1]
RBRACK[159,1]
INT(8)[159,2]
ID(nilnil840)[160,1]
TYPE_STRING[161,1]
SEMICOLON[161,7]
LBRACK[161,8]
SEMICOLON[161,9]
SEMICOLON[161,10]
RBRACK[161,11]
TYPE_STRING[161,12]
ID(array7527)[163,1]
ID(whilearray7765)[164,1]
STRING("vMeClZ")[165,1]
LBRACE[165,9]
EQ[170,3]
LT[170,4]
RBRACK[170,5]
RETURN[170,6]
ASSIGN[170,12]
LBRACK[170,14]
LBRACK[170,15]
ID(whilei7530)[170,16]
ID(arrayk8DjvL8615)[172,1]
INT(3)[173,1]
CLASS[174,1]
PLUS[174,6]
GT[174,7]
LT[174,8]
RBRACK[174,9]
LBRACK[174,10]
GT[174,12]
INT(1642)[174,13]
RETURN[175,1]
STRING("FN")[175,7]
ID(Jnew)[175,11]
RPAREN[175,15]
TYPE_INT[175,23]
RBRACK[175,26]
INT(3)[175,27]
DOT[177,1]
LT[177,2]
INT(8)[177,4]
ID(extends4089)[178,1]
GT[179,1]
ID(Ye)[179,2]
COMMA[179,4]
MINUS[179,5]
ID(intwhileint)[179,6]
ASSIGN[179,17]
STRING("")[179,19]
RBRACK[179,21]
LBRACE[179,22]
COMMA[179,39]
NEW[179,40]
RPAREN[179,43]
LBRACK[179,44]
DOT[179,45]
WHILE[179,46]
STRING("")[179,51]
INT(354)[179,53]
LBRACE[180,1]
GT[180,2]
LT[180,3]
MINUS[180,4]
INT(36)[180,5]
ID(extendsstring)[181,1]
PLUS[181,19]
ID(voidintnew)[181,20]
LPAREN[181,30]
MINUS[181,31]
ID(while946)[181,32]
ID(returnnew)[182,1]
LBRACE[182,10]
INT(420)[182,11]
TYPE_VOID[183,1]
COMMA[183,5]
EXTENDS[183,6]
SEMICOLON[183,13]
LBRACK[183,14]
ID(whileintnew)[183,15]
ASSIGN[183,26]
ID(nil87)[183,28]
EXTENDS[184,2]
MINUS[184,10]
IF[184,11]
RPAREN[184,13]
EXTENDS[184,14]
LPAREN[184,21]
RBRACE[184,22]
IF[184,23]
GT[184,25]
ID(classnew)[184,26]
ID(nilHEclass)[184,35]
LPAREN[184,45]
NEW[184,46]
RPAREN[184,49]
ID(intwhile2483)[184,50]
INT(727)[185,1]
ID(nil9229)[186,1]
EQ[187,1]
ID(stringextendsarraywhile)[187,2]
LT[187,25]
EQ[187,26]
ID(gclassnil)[187,27]
LPAREN[187,36]
MINUS[187,37]
RPAREN[187,38]
CLASS[187,39]
LBRACK[187,44]
ARRAY[187,45]
DOT[187,50]
EQ[187,51]
RBRACE[187,52]
MINUS[187,53]
LPAREN[187,54]
ID(yq433)[187,55]
COMMA[188,1]
TYPE_VOID[188,2]
MINUS[188,6]
LBRACK[188,7]
ID(voidextendsarray)[188,8]
LBRACK[188,24]
COMMA[189,6]
ID(extendsint)[189,7]
LBRACE[189,17]
LT[189,18]
ID(classvoidreturn)[189,19]
SEMICOLON[189,34]
DOT[189,35]
TYPE_STRING[189,36]
LBRACK[189,42]
ASSIGN[189,43]
INT(70)[189,45]
PLUS[192,1]
RBRACK[192,2]
STRING("")[192,3]
ID(ifnil)[192,5]
MINUS[192,10]
ASSIGN[192,11]
CLASS[192,13]
RBRACE[192,18]
INT(2007)[192,19]
INT(683)[193,1]
GT[194,2]
INT(777)[194,3]
ID(voidwhile)[195,1]
LBRACE[195,10]
PLUS[195,11]
INT(8)[195,12]
TYPE_STRING[196,1]
STRING("jG")[196,7]
ID(extends12)[196,11]
ID(z33int98)[197,1]
DOT[198,1]
ID(Q7rjxI348)[198,2]
EXTENDS[199,1]
RBRACK[199,8]
COMMA[199,9]
EQ[199,10]
ASSIGN[199,11]
ID(classvoidextendsa)[199,13]
STRING("v")[202,1]
RBRACK[202,4]
STRING("JJ")[202,5]
WHILE[202,9]
LPAREN[202,14]
LT[202,15]
TYPE_INT[202,16]
ASSIGN[202,19]
MINUS[203,1]
INT(60)[203,7]
SEMICOLON[204,1]
WHILE[204,2]
EQ[205,1]
ID(arraystring)[205,2]
LT[205,13]
INT(172)[205,14]
SEMICOLON[206,1]
ID(intclassreturn)[206,2]
RPAREN[206,16]
EXTENDS[206,17]
ID(x2VF)[206,25]